metric_source/local/errors.go

```diff
@@ -33,7 +34,7 @@ func (err ErrUnknownFunction) Error() string {
 
 // isErrUnknownFunction checks error for carbonapi.errUnknownFunction
 func isErrUnknownFunction(err error) bool {
-	switch err.(type) {
+	switch merry.Unwrap(err).(type) {
```

Comment:
missed any tests on this case?

let's add?

Reply:
This change was caused by package update. Without it trigger tests start to fail